src/main/resources/data/invoices/19284_paid.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": "50129fe1-ca11-43be-a2dd-e1adfb0ead29",
+  "adjustments": [
+  ],
+  "adjustmentsTotal":10.00,
+  "approvedBy": "b72a28f6-04fc-489f-be18-ad53a3a64f67",
+  "approvalDate": "2018-07-50T00:00:00Z",
+  "currency": "USD",
+  "folioInvoiceNo": "19284",
+  "invoiceDate": "2018-07-01T00:00:00Z",
+  "paymentMethod": "EFT",
+  "status": "Paid",
+  "source": "024b6f41-c5c6-4280-858e-33fba452a334",
+  "subTotal": 342.00,
+  "total": 352.00,
```

Comment:
PO with id="efff3ffe-5469-1f8f-aa3a-7afc933f9eff" this invoice is based on has 3 PO Lines with estimated total `734.37`.

Reply:
I see, it is covering only one PO Line and 1 reseource of 2 ordered.